### DISCUSSION TITLE
Allow use of tags over 50 characters long

### DISCUSSION THREAD

**Discussion by lmuhlha** [grafana/grafana#37019] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Allow use of tags over 50 characters long

**Why is this needed**:
We use tags in the format of `component-id: service-name`, for example, and some service names can be long. This results in 500 responses from Grafana if the tag ends up being over the 50 character limit.
We feel this limit is fairly low to begin with.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
